Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -7,9 +7,27 @@ public static class ArrayOperations
         /// If array empty or null- don't do anything.
         /// </summary>
         /// <param name="array">Input array in a random order.</param>
+        ///
+        private static void ExchangeData(int[] array, int i, int j)
+        {
+            int temp = array[i];
+            array[i] = array[j];
+            array[j] = temp;
+        }
+
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array == null) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887369)